packages/hooks/use-media-query/tests/index.test.ts

```diff
@@ -0,0 +1,12 @@
+import { renderHook } from "@yamada-ui/test"
+import { useMediaQuery } from "../src"
+
+describe("useMediaQuery", () => {
+  test("`(prerers-color-scheme: dark)` should be truthy", () => {
```

Comment:
Should this `(prerers-color-scheme: dark)` be `(prefers-color-scheme: dark)`?

Reply:
@setodeve 

I'm sorry. I fixed a typo.